siphon/simplewebservice/iastate.py

```diff
@@ -14,7 +14,7 @@
 from .._tools import get_wind_components
 from ..http_util import HTTPEndPoint
 
-warnings.filterwarnings('ignore', 'Pandas doesn\'t allow columns to be created', UserWarning)
+warnings.filterwarnings('ignore', "Pandas doesn\'t allow columns to be created", UserWarning)
```

Comment:
The double quotes is to make flake8 pass. I'll try it without the escape and see if it's still happy.

Reply:
Right, I think flake8 wants the double quotes because there's a quote in there that has to be escaped because of it. Honestly, removing the escape is the only benefit to shifting.